src/Call/Oneclick/InitPaymentRequest.php

```diff
@@ -45,11 +43,11 @@ public function __construct(
 		string $merchantId,
 		string $origPayId,
 		string $orderId,
-		int $totalAmount = null,
-		Currency $currency = null,
+		Cart $cart = null,
```

Comment:
@pepakriz pro mě amount nijak neevokuje to, že by to mělo něco společnýho konkrétně s penězi, protože amount může být i v kilogramech nebo čímkoliv jiným, takže proto by mi dávalo smysl něco jako Price nebo Money nebo tak :) Ale to jejich API pořádně neznám - není to ale tak, že amount používají technicky pro to, co je v té tvé class to value? Jakože amount + currency = price, to by mi dávalo smysl :)